pages/common/vegeta.md

```diff
@@ -1,6 +1,6 @@
 # vegeta
 
-> Vegeta is both a command line utility and a library for HTTP load testing.
+> A command line utility and a library for HTTP load testing.
```

Comment:
This change was originally suggested by @sbrl [here](https://github.com/tldr-pages/tldr/pull/2519#discussion_r230943663), but it didn't seem to be included in the merged page (it wasn't argued against either -- I suppose it simply slipped by?)